programs/test/metatest.c

```diff
@@ -46,6 +46,12 @@
  */
 volatile int false_but_the_compiler_does_not_know = 0;
 
+/* Hide calls to calloc/free from static checkers such as
+ * `gcc-12 -Wuse-after-free`, to avoid compile-time complaints about
+ * code where we do mean to cause a runtime error. */
+void * (* volatile calloc_but_the_compiler_does_not_know)(size_t, size_t) = mbedtls_calloc;
+void(*volatile free_but_the_compiler_does_not_know)(void *) = mbedtls_free;
```

Comment:
Surely one of these is going to annoy the code style check?

Reply:
Apparently not, but can we standardise on one style?

Reply:
[This feels inconsistent to me too](https://arm-ce.slack.com/archives/GTM3SM1K5/p1700570538230139), but I'm not going to expend energy to improve the presentation of function pointer declarations, which come up very rarely in our code base.

Reply:
Oh. Right. I misunderstood, uncrustify is actually **enforcing** this, not being lax about it. In which case nvm